README.md

```diff
@@ -12,9 +12,6 @@
 <a href="LICENSE">
     <img src="http://img.shields.io/badge/license-MIT-brightgreen.svg" alt="MIT License">
 </a>
-<a href="https://github.com/vapor/postgres-nio/actions">
-    <img src="https://github.com/vapor/postgres-nio/workflows/test/badge.svg" alt="Continuous Integration">
-</a>
```

Comment:
```html
<a href="https://github.com/vapor/postgres-nio/actions/workflows/main-codecov.yml">
    <img src="https://github.com/vapor/postgres-nio/actions/workflows/main-codecov.yml/badge.svg?branch=main&event=push" alt="Continuous Integration">
</a>
```
The badge will reflect the status of the most recent code coverage update run made against `main`, which happens whenever a PR is merged - that's the most accurate representation of "CI status" there is for the current CI setup. (And, of course, I have major plans around improving the CI across the board anyway 🙂)

Reply:
(Oh, for reference, since I had to hunt it down again myself just now - to get the badge URL for a workflow, go to the Actions tab, select the workflow on the left, and click the ellipsis "…" button to the right of the search field to get a dropdown with a "Create status badge" option. So easy to find! 🙃)